### ISSUE TITLE
sql: parens in group by expression affect the columns you can project

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/cockroach#10145]
When grouping by multiple cols, if you put parens around them, then you can't project individual cols any more. Not sure how the parens matter; they probably trick compilation into thinking we're grouping by a general expression. FWIW it works in postgres.

```
root@:26257> SELECT k FROM t.t GROUP BY (k);
pq: column "k" must appear in the GROUP BY clause or be used in an aggregate function
```

And without parens:

```
root@:26257> SELECT k FROM t.t GROUP BY k;
+---+
| k |
+---+
| 1 |
| 2 |
+---+
(2 rows)
```

**Comment by knz** [comment]
The problem is that the group by expressions are not normalized.

**Comment by knz** [comment]
Fixed by 146344b799cb94f42789336bb156c614a06be9d5 I think, definitely fixed in d40c7b7c1f68856791f65d34859a127121cffa31.
